.github/workflows/pull-request.yml

```
@@ -253,6 +253,54 @@ jobs:
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
 
+  Docker-Start-Check:
+          name: Check if Talawa Admin app starts in Docker
+          runs-on: ubuntu-latest
+          needs: [Code-Quality-Checks]
+          steps:
```

actionlint

could not parse as YAML: yaml: line 260: did not find expected '-' indicator

(syntax-check)